lab/lab/StopLight/LightView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  LightView.swift
+//  lab
+//
+//  Created by Sam Franusic on 6/10/24.
+//
+
+import SwiftUI
+
+struct LightView: View {
+    @StateObject var model: LightViewModel
+
+    var body: some View {
+        Circle()
+            .fill(
+                model.isOn ? model.lightColor : .black
+            )
+            .animation(/*@START_MENU_TOKEN@*/.easeIn/*@END_MENU_TOKEN@*/(duration: 0.5), value: model.isOn)
+    }
+}
+
+#Preview {
+    LightView(model: LightViewModel(.green))
```

Comment:
this does not work for the preview since the color only shows when the LigtViewModel.isOn = true, but by default is alway false

Reply:
Good point. Would you recommend using a mock model in the preview, or changing the actual model to be on by default?

Reply:
if you want to  go for simplicity it can be  removed, since by default it would be black, if you want to go for completion, a turned off or default state in the view model would make sense

Reply:
Done. Thanks! Appreciate the review